### PR TITLE
feat(collect-data-modal): apply real data

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### Things to Talk About
Plz check by commits

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue [hunk 1]
```diff
@@ -66,55 +83,114 @@
...
+
+/* API */
+const getJobProgress = async () => {
+    try {
+        const response = await SpaceConnector.client.inventory.job.getJobProgress({
+            job_id: collectorDataModalState.recentJob.job_id,
+        });
+        state.jobTaskStatus = response.job_task_status;
+    } catch (e) {
+        ErrorHandler.handleError(e);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:175]: suggestion: 
```typescript
state.jobTaskStatus = {
    succeeded: 0,
    failed: 0,
    total: 0,
}
```

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue [hunk 2]
```diff
@@ -66,55 +83,114 @@
...
+    }
+};
+
+const getJobLists = async () => {
+    try {
+        const response = await SpaceConnector.client.inventory.job.list({ query: apiQueryHelper.data });
+        const item = response.results[0];
+        state.duration = durationFormatter(item.created_at, dayjs(), storeState.timezone) || '--';
+    } catch (e) {
+        ErrorHandler.handleError(e);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:185]: suggestion: here, too!
```typescript
state.duration = ...
```

**Review by yuda110 — APPROVED** [review]
LGTM!!! Very clear code!!!!

**Review by sulmoJ — APPROVED** [review]
LGTM!!

**Pull request merged**